web/themes/custom/digital_gov/templates/partials/featured-collection.html.twig

```diff
@@ -0,0 +1,44 @@
+{#
+# Displays a list of featured resources
+
+EXPECTED VARIABLES:
+  - heading: boolean
+  - list: array (see nested variables)
+    - link: string url
+    - title: string or renderable markup
+    - summary: string or renderable markup
+    - deck: string or renderable markup
+#}
+
+{% if list|length %}
+  <div class="featured-collection">
+    {% if heading %}
+      <h2>{{ heading }}</h2>
+    {% endif %}
+    <ul>
+      {% for item in list %}
+        <li>
+          <div class="icon">
+            <img
+              src="/{{ active_theme_path() }}/static/digitalgov/img/digit-50.png"
+              alt=""
+              aria-hidden="true"
+              width="16"
+              height="16"
+            />
+          </div>
+          <div class="copy">
+            <p>
+              <a href="{{ item.link }}" title="{{ item.title }}">
+                <span>{{ item.title }}</span>
+              </a>
+              {% if item.summary or item.deck %}
+                — {{ item.summary|default(item.deck) }}
+              {% endif %}
```

Comment:
on digital.gov, very long summaries seem to be cut off around 140 chars:

![image](https://github.com/user-attachments/assets/1164a985-17b5-4981-ab38-1c84130255a0)

Also there are no spaces between the closing a tag, dash, and summary which renders the title a bit more compactly (Not sure if this makes a difference)

![image](https://github.com/user-attachments/assets/47cce25b-9bad-4306-bd91-4d30aa7fb9e6)

Reply:
ignore the yello above, the `</a>-Learn` is the spacing difference.

This line in the hugo template is truncating long summaries:

```
{{- .Params.summary | markdownify | truncate 140 -}}
```

Reply:
All these revisions are captured in the latest commits